packages/dendron-cli/src/commands/launchEngineServer.ts

```diff
@@ -101,9 +106,36 @@ export class LaunchEngineServerCommand extends CLICommand<
       vaults,
       ws: wsRoot,
     });
+
     if (init) {
       this.L.info({ ctx, msg: "pre:engine.init" });
       const out = await engine.init();
+
+      // These events will only upload if the upload action completes before the
+      // CLI command completes. They are uploaded on a best effort basis.
+      engine.onEngineNoteStateChanged((entries) => {
```

Comment:
fine for now but in the future, if we find this happens a lot - we might want to build a system around writing these metrics to disk and having a background thread that uploads entries when it can basis